src/PieChart/PieArcSeries/PieArc.tsx

```diff
@@ -20,9 +21,11 @@ export interface PieArcProps {
   tooltip?: ReactElement<ChartTooltipProps, typeof ChartTooltip> | null;
   cursor?: string;
   disabled?: boolean;
-  onClick?: (e) => void;
+  onClick?: (e: PieArcClickEvent) => void;
   onMouseEnter?: (e) => void;
   onMouseLeave?: (e) => void;
+  icon?: ReactElement<PieArcIconProps> | null;
```

Comment:
Good. I'll use formatter. It also will resolve the question with `defaultRenderIcon` function

Reply:
I've decided to split arc icons into a separate PR to reduce the scope of the current PR (custom labels which are rendered outside of arc). It will be easier to discuss and finalize an implementations.